app/views/install.scala.html

```diff
@@ -8,7 +8,7 @@
                 <h1><i class='fa fa-cloud-download'></i> Installation</h1>
                 @includes.advertisement()
                 <article><p>Installing SDKMAN! on UNIX-like platforms is as easy as ever.
-                    SDKMAN! installs smoothly on Mac OSX, Linux, Cygwin,
+                    SDKMAN! installs smoothly on Mac OSX, Linux, Cygwin, WLS,
```

Comment:
I'd even put WLS before Cygwin, wdyt?

Reply:
Works for me. AFK right now. Go ahead and do it for me if you have time.